nano/node/network.cpp

```diff
@@ -769,7 +769,7 @@ void nano::network::ongoing_cleanup ()
 {
 	cleanup (std::chrono::steady_clock::now () - node.network_params.network.cleanup_cutoff ());
 	std::weak_ptr<nano::node> node_w (node.shared ());
-	node.workers.add_timed_task (std::chrono::steady_clock::now () + node.network_params.network.cleanup_period, [node_w] () {
+	node.workers.add_timed_task (std::chrono::steady_clock::now () + std::chrono::seconds (1), [node_w] () {
```

Comment:
What is happening to cleanup_period, are we dropping its use?
Isn't checking all sockets every second a bit excessive?

Reply:
The cleanup period was set to 60 seconds by default and is still used in multiple different contexts, with interdependencies not being very clear. Here we just iterate all sockets (~1000 max by default) every second to see if any of them are dead. In the grand scheme of things this iteration is a small blip on the profiler chart.

Reply:
But 1 second might be a bit excessive, adjusting it to 5 seconds should not negatively impact the fixes in any way.